examples/smtp-client/README.md

```diff
@@ -0,0 +1,5 @@
+# SMTP client example
+
+This example shows how to send emails via Gmail.
+Before running this example, open main.c and modify settings at the top
```

Comment:
Thought the new OAUTH policy also applied to SMTP. Great if it doesn't

Reply:
At least this code works well.
Also the same Gmail SMTP service is used for subscription to our cloud services, vcon.io and mdash.net - so yeah, it works so far.